src/api.js

```diff
@@ -0,0 +1,49 @@
+export const getShowById = async (id) => {
```

Comment:
![review](https://user-images.githubusercontent.com/26338350/137945522-9f52dae7-240a-4a1c-99aa-f32faad792cd.png)

- [ ] kindly make the bottom of the elements be in the same line.